tests/Pregel/DurationTest.cpp

```diff
@@ -0,0 +1,60 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2021-2021 ArangoDB GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Markus Pfeiffer
+////////////////////////////////////////////////////////////////////////////////
+
+#include <gtest/gtest.h>
+#include <variant>
+
+#include <fmt/core.h>
+
+#include <Pregel/Status/ExecutionStatus.h>
+
+using namespace arangodb::pregel;
+
+struct DurationTest : ::testing::Test {};
+TEST_F(DurationTest, test_duration) {
```

Comment:
Out of curiosity: Why is this a fixture? There is just one test, so you could just use TEST(DurationTest, test_duration)

Reply:
Dumb question: what is the difference? I just blindly copied some other test.

Reply:
I thought fixtures are feature of test-frameworks to use the same data for several tests. The fixture can have a SetUp function that is called at the start of each test where the fixture is used and a TearDown function which is called at the end of each of these tests.
With `TEST_F` you define which fixture to use by giving the Test Suite name (the first parameter of `TEST_F`) the same name as the fixture struct - here DurationTest - and then `TEST_F` calls the SetUp and TearDown functions automatically when this test runs.
I the code you don't have any setup - actually you just have one test - therefore the struct is irrelevant and you can just use the normal `TEST` macro instead of `TEST_F`.
In my opinion fixtures often make the tests harder to read, but if your setup is complicated they can be useful. Additionally, in my opinion it is strange that googletest requires that the TestSuiteName - here DurationTest - has to be the Fixture name.